website/sidebars.js

```diff
@@ -47,18 +49,47 @@ const sidebars = {
       "label": "Dashboard",
       "type": "category"
     },
-    {
+
+    ],
+    "Intelligence Kit": [
+      "intelligence-kit/overview-intel",
+{
+        "items": [
+          "brain/brain",
+          "intelligence-kit/brain/brain-v2",
+        ],
+        "label": "Brain",
+        "type": "category"
+
+      },
+    ],
+
+    "Development Tools": [
+      {
         "items": [
             "debug_cable/debug-cable",
-            "debug_cable/service-cable",
+            "debug_cable/service-cable"
       ],
       "label": "Cables",
       "type": "category"
     },
-      "mcu_kit/microcontroller-kit",
-      "brain/brain",
-      "pendant/pendant",
+    {
+      "items": [
+        "pendant/pendant"
+
+      ],
+      "label": "Pendant",
```

Comment:
Pendant should not be in the `Development Tools` category. Should be in `Amiga Rover` IMO